rhel7/CMakeLists.txt

```diff
@@ -8,7 +8,7 @@ set(DISA_SRG_VERSION "v1r4")
 set(DISA_STIG_VERSION "v1r1")
 set(DISA_SRG_TYPE "os")
 
-ssg_build_product(${PRODUCT})
+ssg_build_product(${PRODUCT} ${DISA_STIG_VERSION})
```

Comment:
I am assuming a no here, but do we see any adverse affects by adding this?

Reply:
We would have to pass the parameter for every product, not only the ones that need it. But I have no other idea if I want to add the references to the unlinked xccdf file.

Do you think of another approach to it?